packages/plugins/minos-http-aiohttp/minos/plugins/aiohttp/requests.py

```diff
@@ -90,14 +90,14 @@ def user(self) -> Optional[UUID]:
             return None
         return UUID(self.headers["user"])
 
-    @property
+    @cached_property
     def headers(self) -> dict[str, str]:
         """Get the headers of the request.
 
         :return: A dictionary in which keys are ``str`` instances and values are ``str`` instances.
         """
         # noinspection PyTypeChecker
-        return self.raw.headers
```

Comment:
Could you add a test that checks if `headers` attribute can be modified?

Reply:
Done!